src/components/QRDialog.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div class="qrDialog__content">
+		<h2>{{ title }}</h2>
+		<div>
+			<img :src="uri"
+				:title="text"
+				:alt="t('forms', 'QR code representation of {text}', { text: text })">
+		</div>
+	</div>
```

Comment:
```suggestion
	<NcDialog class="qr-dialog"
		close-on-click-outside
		:name="title"
		:open="open"
		@close="$emit('close')>
		<div class="qr-dialog__content">
			<img :src="uri"
				:title="text"
				:alt="t('forms', 'QR code representation of {text}', { text: text })">
		</div>
	</NcDialog>
```

The idea is to wrap the dialog not only the content, especially to use the NcDialog `name` instead of custom heading.

Reply:
Yes, it definitely had some refactoring due, like missing quotes ;) .
Was more struggling with opening and closing and reopening of NcDialog though.